scripts/pelorus-operator-patches/07_spec_description.diff

```diff
@@ -0,0 +1,89 @@
+           kind:
+-            description: 'Kind is a string value representing the REST resource this
+-              object represents. Servers may infer this from the endpoint the client
+-              submits requests to. Cannot be updated. In CamelCase. More info: https://git.k8s.io/community/contributors/devel/sig-architecture/api-conventions.md#types-kinds'
++            description: >-
++              Kind is a string value representing the REST resource this
++
++              object represents. Servers may infer this from the endpoint the
++              client
++
++              submits requests to. Cannot be updated. In CamelCase. More info:
++              https://git.k8s.io/community/contributors/devel/sig-architecture/api-conventions.md#types-kinds
+             type: string
+           metadata:
+             type: object
+           spec:
+-            description: Spec defines the desired state of Pelorus
++            description: >-
++              PelorusSpec defines state of the Pelorus application and allows
++              to configure for the desired workflow. More information about
++              Pelorus configuration that can be used in the Spec section is
++              available at:
++              https://pelorus.readthedocs.io/en/latest/Configuration/
+             type: object
+-            x-kubernetes-preserve-unknown-fields: true
++            properties:
++              exporters:
++                description: >-
++                  References configuration for the Pelorus exporters. More
++                  info about exporters configuration:
++                  https://pelorus.readthedocs.io/en/latest/Configuration/#configuring-exporters-details
++
++                    Example:
++                      instances:
++                        - app_name: < instance_name >
++                          enabled: < won't be created if set to false >
++                          exporter_type: < deploytime, committime, failure >
++                type: object
++              prometheus_retention:
++                description: >-
++                  Prometheus Retention time. More information:
++                  https://pelorus.readthedocs.io/en/latest/Install/#configure-prometheus-retention
++                type: string
++              prometheus_retention_size:
++                description: >-
++                  Prometheus Retention Size. More information:
++                  https://pelorus.readthedocs.io/en/latest/Install/#configure-prometheus-retention
++                type: string
++              prometheus_storage:
++                description: >-
++                  Use Prometheus Persistent Volume. More information:
++                  https://pelorus.readthedocs.io/en/latest/Install/#configure-prometheus-persistent-volume-recommended
++                type: boolean
++              prometheus_storage_pvc_capacity:
++                description: Prometheus Persitent Volume capacity.
```

Comment:
TYPO in "persistent"? (missing second S) (also in line 80)

Reply:
Fixed